README.md

```diff
@@ -157,6 +206,31 @@ The journey of a dataset throughout various data transformations and pipelines.
 even if it involves multiple applications or ETL pipelines.
 
 
+## Usage
+
+### Atum Agent routines
+
+TBD
+
+### Control measurement types
+
+The control measurement of a column is a hash sum. It can be calculated differently depending on the column's data type and
+on business requirements. This table represents all currently supported measurement types:
+
+| Type                               | Description                                                   |
+|------------------------------------|:--------------------------------------------------------------|
+| AtumMeasure.RecordCount            | Calculates the number of rows in the dataset                  |
+| AtumMeasure.DistinctRecordCount    | Calculates DISTINCT(COUNT(()) of the specified column         |
+| AtumMeasure.SumOfValuesOfColumn    | Calculates SUM() of the specified column                      |
+| AtumMeasure.AbsSumOfValuesOfColumn | Calculates SUM(ABS()) of the specified column                 |
+| AtumMeasure.SumOfHashesOfColumn    | Calculates SUM(CRC32()) of the specified column               |
+| Measure.UnknownMeasure             | Custom measure where the data are provided by the application |
+
+[//]: # (| controlType.aggregatedTruncTotal    | Calculates SUM&#40;TRUNC&#40;&#41;&#41; of the specified column       |)
```

Comment:
these are todo?

Reply:
These are functions just last week implemented in Atum. We will need to add them to Atum service too. As I was copying the whole block of the function description I took this one too.

Reply:
okay